jdaviz/configs/default/plugins/subset_plugin/utils.py

```diff
@@ -0,0 +1,175 @@
+# utility functions for subset plugin
+# these don't belong in glue.core.region_translators, as they deal with conversions that
+# only happen within this plugin (e.g between region/'subset defintion')
+
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from jdaviz.core.region_translators import regions2roi
+import numpy as np
+from regions import CircleAnnulusSkyRegion, CircleSkyRegion, EllipseSkyRegion, RectangleSkyRegion
+
```

Comment:
Do not want to accidentally bubble up the private functions here up.
```suggestion

__all__ = []
```